lib/yggdrasil/command/move.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Yggdrasil.Command.MoveCommand do
+  defstruct exit: nil
+
+  defimpl Yggdrasil.Command, for: Yggdrasil.Command.MoveCommand do
```

Comment:
The only thing I see is this, but probably not worth holding this up, you can drop the `, for: ...` portion so it looks like this
```elixir
defimpl Yggdrasil.Command do
```
since it's in the module.

That said I'm perfectly ok merging it, just saw it.